src/core/src/pass/sdpa_to_paged_attention.cpp

```diff
@@ -53,8 +53,21 @@ bool ov::pass::SDPAToPagedAttention::run_on_model(const std::shared_ptr<ov::Mode
 
     auto sliding_window = v0::Constant::create(element::i32, Shape{}, {0});  // sliding_window
 
+    auto has_parameter = [=](const std::shared_ptr<ov::Model>& model, const std::string& name) -> bool {
```

Comment:
Tests?

Reply:
@rkazants, tests for the lambda? I didn't add it, just moved up

Reply:
come on, you know what I am taking about:)
I meant tests for the whole PR.

Reply:
you can use `katuni4ka/tiny-random-minicpmv-2_6`